modules/microshift-lvms-system-requirements.adoc

```diff
@@ -11,9 +11,10 @@ Using LVMS in {microshift-short} requires the following system specifications.
 [id="lvms-volume-group-name_{context}"]
 == Volume group name
 
-If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command. 
-* {microshift-short} assigns a default VG when only one VG is found. 
-* If more than one VG is present, the VG named `microshift` is assigned as the default. 
+If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command.
```

Comment:
Might want to consider defining LVMS in the first usage.

```suggestion
If you did not configure LVMS in an `lvmd.yaml` file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command.
```

Reply:
This is not part of the changes made in this jira. 
Also should "YAML" be removed in line 14?